src/Makefile.am

```diff
@@ -1,7 +1,7 @@
 ## Process this file with autoconf to produce configure.
 
-AM_CFLAGS   = -Wall -Wwrite-strings $(PTHREAD_CFLAGS)
-AM_CXXFLAGS = -Wall -Wwrite-strings -Woverloaded-virtual $(PTHREAD_CFLAGS)
+AM_CFLAGS   = -Wall -Wwrite-strings $(PTHREAD_CFLAGS) -fgnu-tm
```

Comment:
We have to handle error
cc1: error: unrecognized command line option "-fgnu-tm"
on unsupported platforms.